DOCS.md

```diff
@@ -124,6 +124,26 @@ pipeline:
     root_dir: some/path/here
 ```
 
+## targets
+You may want to only target a specific list of resources within your terraform code. To achieve this you can specify the `targets` parameter. If left undefined all resources will be planned/applied against as the default behavior. You can define multiple targets with a comma separated list.
+
+```yaml
+pipeline:
+  terraform:
+    image: jmccann/drone-terraform:0.5
+    plan: false
+    targets: aws_security_group.generic_sg, aws_security_group.app_sg
```

Comment:
You should wrote a single string or an array in yaml format. Those are the options we also use for other drone plugins. We don't use a comma separated list anywhere